crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -99,16 +99,23 @@ def health_check
       end
 
       def ceph_status
-        {}.tap do |ret|
-          ceph_node = ::Node.find("roles:ceph-mon AND ceph_config_environment:*").first
-          next unless ceph_node
-          ssh_retval = ceph_node.run_ssh_cmd("LANG=C ceph health 2>&1")
-          unless ssh_retval[:stdout].include? "HEALTH_OK"
-            ret["errors"] = [
-              "ceph cluster health check failed with #{ssh_retval[:stdout]}"
-            ]
-          end
+        ret = {}
+        ceph_node = ::Node.find("roles:ceph-mon AND ceph_config_environment:*").first
+        return ret if ceph_node.nil?
+
+        ssh_retval = ceph_node.run_ssh_cmd("LANG=C ceph health 2>&1")
```

Comment:
This probably wants to be `LANG=C ceph --connect-timeout 5 health 2>&1"`; that way it'll wait five seconds to get the status then die. Otherwise if the MONs aren't quorate, this command will hang forever.